resources/rgbscripts/alternate.js

```diff
@@ -86,7 +89,9 @@ var testAlgo;
 
   algo.color1Index = algo.getColorIndex("Red");
   algo.properties.push("name:color1Index|type:list|display:Color 1|" +
-    "values:" + colorPalette.names.toString() + "|" +
+    "values:" + colorPalette.names.toString() +
+//    ",Matrix RGB" +
```

Comment:
this doesn't really seem like a needed change, is it?

Reply:
Yes, this is leftover and should not be there. I'll remove it.